tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/sync.py

```diff
@@ -100,7 +100,7 @@ class LoadManifest(Step):
     def create(self, state):
         from manifest import manifest
         state.manifest_path = os.path.join(state.metadata_path, "MANIFEST.json")
-        state.test_manifest = manifest.Manifest("/")
+        state.test_manifest = manifest.Manifest(state.sync["path"], "/")
```

Comment:
Is this different to `state.tests_path`?

Reply:
I guess we use it with `manifest.update` so it seem fine.

Reply:
When I used state.tests_path, I got exceptions for files that existed in the sync tree and not in the pre-sync tree.